assistants/prospector-assistant/assistant/chat.py

```diff
@@ -114,22 +114,11 @@ async def on_message_created(
     await legacy.provide_guidance_if_necessary(context)
 
 
-is_doc_agent_running = False
-
-
 @assistant.events.conversation.message.command.on_created
 async def on_command_message_created(
     context: ConversationContext, event: ConversationEvent, message: ConversationMessage
 ) -> None:
-    config = await assistant_config.get(context.assistant)
-    metadata: dict[str, Any] = {"debug": {"content_safety": event.data.get(content_safety.metadata_key, {})}}
-
-    # config.agents_config.document_agent.enabled = True  # To do... tie into config.
-    global is_doc_agent_running
-    is_doc_agent_running = True
-
-    doc_agent = DocumentAgent(attachments_extension)
-    await doc_agent.receive_command(config, context, message, metadata)
+    pass
```

Comment:
If we're just going to pass here, consider just deleting this handler until if/when we want it.